### ISSUE TITLE
Crossbow, and COD flashbacks

### ISSUE THREAD

**Issue by silent93x** [Ciastex/ArcadeCoinBugCentral#47]
I don't have words. It's a visual thing, but still, [here](https://youtu.be/66_v0G0jkK0)

**Comment by TieDyeTopHat** [comment]
Not really a bug, I think that's just the ticking damage from one of the AOE attacks?

**Comment by silent93x** [comment]
No, those would beat, this doesn't. it was a bug, and you could see that there was no fire around

**Comment by silent93x** [comment]
no, it's a bug, it stayed there, unfortunately not able to recreate

**Comment by BobTheBob9** [comment]
i *believe* this is an issue with the crosshair not resetting, as i'm sure i've encountered it before. Worth testing whether this occurs from dying/restarting a level or pausing while the hitmarker is on screen.  Could you provide more context as to when this bug actually occured? i.e. what happened around the time it triggered

**Comment by silent93x** [comment]
> i _believe_ this is an issue with the crosshair not resetting, as i'm sure i've encountered it before. Worth testing whether this occurs from dying/restarting a level or pausing while the hitmarker is on screen. Could you provide more context as to when this bug actually occured? i.e. what happened around the time it triggered

What happened is I killed a Derek while dashing into him, got stuck in and hitmarker didn't went away. 
(I think it went away because I mistakenly passed on the same place where Derek died, as you can see in the video)
